Cargo.toml

```diff
@@ -29,7 +29,8 @@ approx      = { version = "0.5", default-features = false }
 decimal     = { version = "2.0", default-features = false, optional = true }
 num-complex = { version = "0.4", default-features = false }
 packed_simd = { package = "packed_simd_2", version = "0.3", features = [ "into_bits" ], optional = true }
-wide        = { version = "0.7", default-features = false, optional = true }
+# wide        = { version = "0.7", default-features = false, optional = true }
+wide = { path = "c:/users/jasper/wide", default-features = false, optional = true }
```

Comment:
that seems like an unintentional change :)

Reply:
Whoops 🤣, updated to remove these changes.